deployment/helm/synopsys-alert/values.yaml

```diff
@@ -47,6 +47,17 @@ cfssl:
 postgres:
   registry: "docker.io/centos" # override the docker registry at container level
   isExternal: false # false for running Postgres as a container and true for using External Postgres database
+  ssl: true # If true, Alert uses SSL for external Postgres connection
+  sslConfig:
+    sslUseFiles: false # If true, Alert will expect to find ssl certs for communicating with the External Postgres database
+    sslKeyFilePath: "" # If using files for ssl, specify ssl key path
```

Comment:
Why would the paths be used over the secrets?  Is this to map to a path in the volume?  I would assume that we just need secrets defined with the certificate contents.

Reply:
I may not have mentioned it, but the cert related settings ask for the file paths not the actual content https://www.postgresql.org/docs/12/libpq-connect.html (search for sslkey). Since it's their paths, should I remove the helm secret functionality?

Reply:
secrets are stored as files.  So you need a path to the secrets file.  That's why I push for secrets.  Secrets are stored as files in a specific directory on the pod.